ui/pages/_app.js

```diff
@@ -594,8 +594,12 @@ class MesheryApp extends App {
                         ) : (
                           <>
                             {' '}
-                            Built with <FavoriteIcon className={classes.footerIcon} /> by the Layer5
-                            Community
+                            Built with{' '}
+                            <FavoriteIcon
+                              style={{ color: '#00ab84' }}
```

Comment:
Thanks for this. Wrong color, sir.

Reply:
Please use a theme-based reference to the "#00b39f" (Keppel green) variable.

Reply:
I have done the required changes suggested by you. Please have a look

Reply:
@VivekBaldha0712 theme reference means don't use directly the hex colors , we have defined theme names for colors under [`themes/app.js`](https://github.com/meshery/meshery/blob/master/ui/themes/app.js) from use a reference called success naming like `theme.palette.seconadary.success`